src/components/ListOrgs.vue

```diff
@@ -272,6 +275,48 @@ const exportAll = async () => {
   exportCsv(exportData, `roar-${activeOrgType.value}.csv`);
 };
 
+const exportOrgUsers = async (orgId) => {
+  try {
+    // Fetch the first page of users to estimate total count
+    const users = await fetchUsersByOrg(activeOrgType.value, orgId.id, ref(10001), ref(0), orderBy);
+
+    if (!users || users.length === 0) {
+      throw new Error('No users found for the organization.');
+    }
+
+    if (users.length >= 10000) {
+      throw new Error('The list is too large to export.');
+    }
+
+    const computedExportData = users.map((user) => ({
+      Username: _get(user, 'username'),
+      Email: _get(user, 'email'),
+      FirstName: _get(user, 'name.first'),
+      LastName: _get(user, 'name.last'),
+      Grade: _get(user, 'studentData.grade'),
+      Gender: _get(user, 'studentData.gender'),
+      DateOfBirth: _get(user, 'studentData.dob'),
+      UserType: _get(user, 'userType'),
+    }));
+
+    exportCsv(computedExportData, `organization-users-${_kebabCase(orgId)}.csv`);
+
+    toast.add({
+      severity: 'success',
+      summary: 'Export Successful',
+      detail: 'Users have been exported successfully!',
+      life: 3000,
+    });
+  } catch (error) {
+    toast.add({
+      severity: 'error',
+      summary: 'Export Failed',
+      detail: error.message,
+      life: 3000,
+    });
+  }
+};
+
```

Comment:
```suggestion
const exportOrgUsers = async (orgType) => {
  try {
    // Fetch the first page of users to estimate total count
    const users = await fetchUsersByOrg(activeOrgType.value, orgType.id, ref(10001), ref(0), orderBy);

    if (!users || users.length === 0) {
      toast.add({
        severity: 'error',
        summary: 'Export Failed',
        detail: 'No users found for the organization.',
        life: 3000,
      });
      return;
    }

    if (users.length >= 10000) {
      toast.add({
        severity: 'error',
        summary: 'Export Failed',
        detail: 'Too many users to export. Please filter the users by selecting a smaller org type.',
        life: 3000,
      });
      return;
    }

    const computedExportData = users.map((user) => ({
      "Username": _get(user, 'username', "(No Username Found)"),
      "Email": _get(user, 'email', "(No Email Found)"),
      "First Name": _get(user, 'name.first', "(No First Name Found)"),
      "Last Name": _get(user, 'name.last', "(No Last Name Found)"),
      "Grade": _get(user, 'studentData.grade', "(No Grade Found)"),
      "Gender": _get(user, 'studentData.gender', "(No Gender Found)"),
      "Date Of Birth": _get(user, 'studentData.dob', "(No Date of Birth Found)"),
      "User Type": _get(user, 'userType', "(No User Type Found)"),
    }));

    // ex. cypress-test-district-users-export.csv
    exportCsv(computedExportData, `${_kebabCase(orgType.name)}-users-export.csv`);

    toast.add({
      severity: 'success',
      summary: 'Export Successful',
      detail: 'Users have been exported successfully!',
      life: 3000,
    });
  } catch (error) {
    toast.add({
      severity: 'error',
      summary: 'Export Failed',
      detail: error.message,
      life: 3000,
    });
  }
};
```
This is looking good! I have a couple of suggestions:

1. `orgType` is an object with properties that we can access
2. Use toasts to inform the user about the state of the export, rather than using `throw` as flow-control
3. Using strings as the column headers allows the .csv export to have spaces in the column names; this is not super critical and probably depends on user preference either way
4. Use the `orgType` properties to make a more descriptive .csv export filename
5. In the future, we might even have a checkbox or button that DOES allow the user to bypass the check and export more than 10,000 users if their browser can handle the computation.

Reply:
Thank you for your suggestions, let me ask P&R if they would like the exporting to contain a default value, or nothing